docs/advanced-customization/custom-widgets-fields.md

```diff
@@ -133,11 +133,10 @@ The following props are passed to custom widget components:
 - `onBlur`: The input blur event handler; call it with the the widget id and value;
 - `onFocus`: The input focus event handler; call it with the the widget id and value;
 - `options`: A map of options passed as a prop to the component (see [Custom widget options](#custom-widget-options)).
+- `options.enumOptions`: For enum fields, this property contains the list of options for the enum as an array of { label, value } objects. If the enum is defined using the oneOf/anyOf syntax, the entire schema object for each option is spread onto the { ...schema.\[anyOf|oneOf\], label, value } object.
```

Comment:
We should clarify that the schema is included regardless of whether it's defined using the oneOf/anyOf syntax.

Reply:
When an enum is declared like

```
{ type: string, enum: [one, two, three] }
```

the options list currently doesn't have a schema defined, they are instead just { label, value } objects, where enumNames might override the label.   

only in cases where not using they enum key would the schema be added.  I think those 2 cases are oneOf/anyOf?   are there other times that's valid?

Reply:
Maybe something along the lines of 

For enum fields defined using the enum property, this contains a list of options as an array of { label, value } objects.  For other enums, such as those defined with an anyOf or oneOf schema, this contains a list of options as an array of { schema, label, value } objects, where the schema is the schema defining each enum option individually

Reply:
On the other hand, we could also pass the base schema for the 

```
{ type: string, enum: [one, two, three] }
```

to the options themselves too, so this might generate

```
{
   schema: { type: string, enum: [one, two, three] },
   label: one,
   value: one
} 
```

for these items as well.  This might make sense since the enum options each inherit that schema?

Reply:
actually, nevermind that, I think it might cause more complexity than it would aim to solve, 

for instance, a 'description' on the parent schema would then automatically apply to the child enums, when it should not - so probably better to avoid adding the schema, or only copy specific schema properties in that case (ie, just type and _maybe_ format, but no others?)

Reply:
Yeah, let's just not pass the base schema for simple (non-oneOf/anyOf enums) instead.